src/baguetteBox.js

```diff
@@ -484,8 +484,15 @@
             return;
         }
 
+        var galleryItem = currentGallery[index];
+        // It's possible that the decision to preload is made,
+        // but by the time the load happens, the element is gone.
+        if (typeof galleryItem  === 'undefined') {
+            return;
+        }
```

Comment:
I'd rename the variable to `galleryImage` and move the whole to the top to become:
```js
var imageContainer = imagesElements[index];
var galleryImage = currentGallery[index];

// Return if the index exceeds prepared images in the overlay
// or if the current gallery has been changed / closed
if (imageContainer === undefined || galleryImage === undefined) {
    return;
}
```
I've also updated the comment. Let me know what do you think.